<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oscyloskop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2" rowspan="3">
       <widget class="QWidget" name="widgetCH" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Wartość minimalna:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="lineEditRms">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Wartość skuteczna:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLineEdit" name="lineEditAmp">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEditMax">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Wartość maksymalna:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLineEdit" name="lineEditFreq">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineEditMin">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Wartość średnia:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Częstotliwość:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="Devices">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Tryb pomiaru:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxCanals"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxMeasure"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Wybór kanału:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Lista Urządzeń:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="ListOfDevices"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Pomiar:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonMeasure">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonStopMeasure">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_6"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Napięcie wyzwolenia:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButtonOff">
           <property name="text">
            <string>Off</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxMax">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Wartość napięcia:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonOn">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QWidget" name="widget2CH" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="2" column="3">
          <widget class="QLineEdit" name="lineEditCH2Max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 127, 14);</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEditCH1Amp">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="15" column="2">
          <widget class="QLineEdit" name="lineEditPApparent"/>
         </item>
         <item row="9" column="3">
          <widget class="QLineEdit" name="lineEditCH2Amp">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 127, 14);</string>
           </property>
          </widget>
         </item>
         <item row="14" column="2">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Moc Pozorna:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditCH1Max">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0" colspan="4">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QLineEdit" name="lineEditCH2Frq">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 127, 14);</string>
           </property>
          </widget>
         </item>
         <item row="15" column="3">
          <widget class="QLineEdit" name="lineEditPReactive"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEditCH1Min">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
           <property name="text">
            <string>CH1:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="lineEditCH1Frq">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Wartość średnia:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 127, 14);</string>
           </property>
           <property name="text">
            <string>CH2:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Moc Czynna:</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="lineEditPActive"/>
         </item>
         <item row="7" column="3">
          <widget class="QLineEdit" name="lineEditCH2Rms">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 127, 14);</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="lineEditCH1Rms">
           <property name="styleSheet">
            <string notr="true">color: rgb(31, 119, 180);</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="lineEditCH2Min">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 127, 14);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Wartość maksymalna:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Wartość maksymalna:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Wartość minimalna:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Wartość minimalna:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Wartość skuteczna:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Wartość skuteczna:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Wartość średnia:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Częstotliwość:</string>
           </property>
          </widget>
         </item>
         <item row="14" column="3">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Moc Bierna:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Częstotliwość:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Zapisz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
